prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -56,10 +73,14 @@ object CustomFHIRFunctions {
     }
 
     /**
-     * Get the function details for a given [functionName].
+     * Get the function details for a given [functionName] and adds the ability to provide [additionalFunctions] in case
+     * additional custom FHIR functions are needed.
```

Comment:
What is the case that we use additional functions in?

Reply:
So we have custom FHIR path functions as part of the FHIR to HL7 conversion library. But, once we move the library to its own repo, we need to have the option to have additional custom FHIR functions outside of the library like the LIVD look up table function. Since that function has logic specific to ReportStream.

Reply:
Awesome, thank you for the explanation!